spec/peep_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'peep'
+
+describe Peep do
+
+  describe '.all' do
+    it 'returns all peeps' do
+      connection = PG.connect(dbname: 'chitter_test')
+      connection.exec("INSERT INTO peeps (message) values ('This is a peep!');")
+
+      peeps = Peep.all
+
+      expect(peeps[0].message).to include 'This is a peep!'
```

Comment:
I think you can make a stronger assertion than that. The value of `message` should be equal to `"This is a peep!"`, not just include it.

Reply:
You might also want to test the `Peep#date` method